constants.py

```diff
@@ -1 +1,20 @@
-defaultkey = "default"
\ No newline at end of file
+defaultkey = "default"
+
+# MSEE Trigger coordinates
```

Comment:
FUTURE: A util script that takes a video, gets the first frame(or whatever reference frame you want), uses cv2.getROI() and adds the trigger coords automatically. As of now if someone wanted to use the repo they would have no idea that nothing will work unless the trigger coords are proper.